chapters/data-collection.tex

```diff
@@ -105,50 +107,65 @@ \subsection{Data licensing agreements}
 and to fully inform others of what you are doing.
 Writing down and agreeing to specific details is a good way of doing that.
 
-When you are licensing someone else's data for research,
-keep in mind that they are not likely to be familiar
+If the research team requires access to existing data for novel research,
+terms of use should be agreed on with the data owner, 
+typically through a data licensing agreement.
+Keep in mind that the data owner is likely not familiar
 with the research process, and therefore may be surprised
 at some of the things you want to do if you are not clear up front.
-You will typically want the right to create and retain
-derivative indicators, and you will want to own that output dataset.
-You will want to store, catalog, or publish, in whole or in part,
-either the original licensed material or the derived dataset.
+You will typically want intellectual property rights to all derivative works developed used the data,
+a license for all uses of derivative works, including public distribution 
+(unless ethical considerations contraindicate this).
+This is important to allow the research team to store, catalog, and publish, in whole or in part,
+either the original licensed dataset or the derived materials.
 Make sure that the license you obtain from the data owner allows these uses,
-and that you check in with them if you have any questions
-about what you are allowed to do with specific portions of their data.
-
-When you are licensing your own data for release,
-whether it is to a particular individual or to a group,
-make sure you take the same considerations.
-Would you be okay with someone else publicly releasing that data in full?
+and that you consult with the owner if you foresee exceptions with specific portions of the data.
+
+If the research team contracts a vendor to collect original data,
+contract terms must clearly stipulate that the research team owns the data
+and maintains full intellectual property rights. 
+The contract should also explicitly stipulate that the contracted firm
+is responsible for protecting the confidentiality of the respondents,
+and that the data collection will not be distributed to any third parties
+or used by the firm or subcontractors for any purpose not expressly stated in the contract,
+before, during or after the assignment. 
+The contract should also stipulate that the vendor is required to comply with 
+ethical standards for social science research, 
+and adhere to the specific terms of agreement with the relevant 
+Institutional Review Board or applicable local authority. 
+
+Research teams that collect their own data must consider the terms
+under which they will release that data to other researchers or to the general public. 
+Will you publicly releasing the data in full (removing personal identifiers))?
 Would you be okay with it being stored on servers anywhere in the world,
 even ones that are owned by corporations or governments abroad?
```

Comment:
What is abroad in this sentence? The US is a [5-eyes](https://restoreprivacy.com/5-eyes-9-eyes-14-eyes/) nation, so it would be weird if it comes across as we are saying "abroad" from an American perspective.

```suggestion
even ones that are owned by corporations or governments in countries other than your own?
```